test/fuzz/transformation_wrap_vector_synonym_test.cpp

```diff
@@ -101,8 +103,6 @@ TEST(TransformationWrapVectorSynonym, BasicTest) {
                OpStore %39 %46
          %48 = OpLoad %6 %8
          %49 = OpLoad %6 %10
-        %100 = OpCompositeConstruct %12 %48 %48
-        %101 = OpCompositeConstruct %12 %49 %49
```

Comment:
I assume that in the end these simply weren't needed? If so then fine, but just wanted to check.

Reply:
This feels odd. The test shouldnt pass without those two lines.

Reply:
Yeah the test indeed failed on this one. Not sure what I did back there.